### PR TITLE
docs: component log display bug versions

### PR DESCRIPTION
**Pull request by kiner-tang**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [X] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

Display bug version info in component changelog.

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | - |
| 🇨🇳 Chinese | - |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

File: .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx [hunk 1]
```diff
@@ -22,6 +42,20 @@ const useStyle = createStyles(({ token, css }) => ({
...
+  bug: css`
+    font-size: 14px;
+    color: #aaa;
+    paddinginlinestart: 4px;
+    &:hover {
+      color: #333;
+    }
+  `,
+  bugList: css`
+    paddingBlock: 5px;
```
Comment (li-jia-nan) [inline .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:54]: 我们一般用 4 的倍数

File: .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx [hunk 2]
```diff
@@ -22,6 +42,20 @@ const useStyle = createStyles(({ token, css }) => ({
   ref: css`
     margin-left: ${token.marginXS}px;
```
Comment (li-jia-nan) [inline .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:43]: 这个可以顺手改成逻辑属性

File: .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx (hunk 1 above)
Comment (li-jia-nan) [reply .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:54]: 干脆用 token 好了

**Comment by afc163** [comment]
标题和图标没对齐。

File: .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx [hunk 3]
```diff
@@ -20,7 +40,22 @@ const useStyle = createStyles(({ token, css }) => ({
   `,
 
   ref: css`
-    margin-left: ${token.marginXS}px;
+    margin-block-start: ${token.marginXS}px;
```
Comment (li-jia-nan) [inline .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:43]: left 好像是 inline-start ?

Comment (kiner-tang) [reply .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:43]: 是的，写错了，我改下

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**